src/server/v2.0/handler/scanexport.go

```diff
@@ -131,8 +131,21 @@ func (se *scanDataExportAPI) DownloadScanData(ctx context.Context, params operat
 		return se.SendError(ctx, err)
 	}
 	execution, err := se.scanDataExportCtl.GetExecution(ctx, params.ExecutionID)
+
 	if err != nil {
-		se.SendError(ctx, err)
+		if notFound := orm.AsNotFoundError(err, "execution with id: %d not found", params.ExecutionID); notFound != nil {
+			return middleware.ResponderFunc(func(writer http.ResponseWriter, producer runtime.Producer) {
+				writer.WriteHeader(http.StatusNotFound)
```

Comment:
return errors.New(nil).WithCode(errors.NotFoundCode).WithMessage("... not found")